ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -402,10 +402,16 @@ private void checkAndMaybeComplete() {
         routerMetrics.operationFailureWithUnsetExceptionCount.inc();
       }
       if (e != null) {
+        // best effort to update metrics if result is available
```

Comment:
does this comment belong here? (or even needed now that the blob ID has the details on encrypted or not)